token/js/src/extensions/transferFee/state.ts

```diff
@@ -52,6 +65,21 @@ export const TransferFeeConfigLayout = struct<TransferFeeConfig>([
 
 export const TRANSFER_FEE_CONFIG_SIZE = TransferFeeConfigLayout.span;
 
+/** Get the fee for given epoch */
+export function getEpochFee(transferFeeConfig: TransferFeeConfig, epoch: bigint): TransferFee {
+    if (epoch >= transferFeeConfig.newerTransferFee.epoch) {
+        return transferFeeConfig.newerTransferFee;
+    } else {
+        return transferFeeConfig.olderTransferFee;
+    }
+}
+
+/** Calculate the fee for the given epoch and input amount */
+export function calculateEpochFee(transferFeeConfig: TransferFeeConfig, preFeeAmount: bigint, epoch: bigint): bigint {
```

Comment:
nit for consistency with the rust side, let's keep the parameters in the same order
```suggestion
export function calculateEpochFee(transferFeeConfig: TransferFeeConfig, epoch: bigint, preFeeAmount: bigint): bigint {
```